keyboards/somei70/somei70.c

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 James Gzowski
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+#ifdef OLED_ENABLE
+static uint8_t  oled_mode       = 0;
+static uint8_t  esc_press_count = 0;
+static uint16_t esc_timer       = 0;
+// Stats display
+void render_stats(void) {
+    char wpm_str[10];
+    snprintf(wpm_str, sizeof(wpm_str), "WPM: %03u", get_current_wpm());
+    oled_write_ln(wpm_str, false);
```

Comment:
I would recommend the `get_u8_str` function. It's much lighter on memory, and for stuff like this, recommended. 
```suggestion
    oled_write_P(PSTR("WPM: "), false);
    oled_write_ln(get_u8_str(get_current_wpm(), '0'), false);
```